src/Essentials/src/Essentials.csproj

```diff
@@ -42,7 +42,9 @@
     <Compile Include="**\*.android.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <Compile Include="**\*.android.*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <AndroidResource Include="Resources\xml\*.xml" />
+    <PackageReference Include="Xamarin.AndroidX.Activity" Version="1.8.2.1" />
     <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.5.0.3" />
+    <PackageReference Include="Xamarin.AndroidX.Collection.Ktx" Version="1.4.0.1" />
```

Comment:
This avoids a `Type androidx.collection.ArraySetKt is defined multiple times` exception -- similar to #20561